src/documents/navIcons.md

```diff
@@ -0,0 +1,215 @@
+```
+
+
+## Grades
+
+This component uses the following base
+[grades](http://docs.fluidproject.org/infusion/development/ComponentGrades.html):
+
+* [`fluid.viewComponent`](http://docs.fluidproject.org/infusion/development/ComponentGrades.html)
+
+## Model
+
+This component supports the following
+[model](http://docs.fluidproject.org/infusion/development/tutorial-gettingStartedWithInfusion/ModelComponents.html)
+properties:
+
+| Path   | Description | Values | Default |
+|--------|-------------|--------|---------|
+| `pageNum` | Corresponds to the index of the current panel. | Panel ID (Number) | 0 |
+| `iconWidth` | The width of the icon. Used to determine where to scroll to make it visible. | Number (in pixels) | 0 |
+
+## Supported Events
+
+This component supports the following
+[events](http://docs.fluidproject.org/infusion/development/InfusionEventSystem.html):
+
+<table>
+    <thead>
+        <tr><th>Event</th><th>Type</th><th>Description</th><th>Parameters</th></tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>`onCreateIcon`</td>
+            <td>default</td>
+            <td>Fired to trigger the creation of each navigation icon</td>
+            <td>
+                <dl>
+                    <dd>`element`</dd>
+                    <dt>the DOM element to use as the container for the [Navigation Icon](icon.md)</dt>
+                    <dd>`position` (number)</dd>
+                    <dt>the position of the [Navigation Icon](icon.md)</dt>
+                </dl>
+            </td>
+        </tr>
+        <tr>
+            <td>`onSetConfirmedIcon`</td>
+            <td>default</td>
+            <td>Fired to trigger a specific panel icon to show confirmed state</td>
+            <td>
+                <dl>
+                    <dd>`panelNum`</dd>
+                    <dt>the number of the panel whose icon state needs to be set to confirmed</dt>
+                </dl>
+            </td>
+        </tr>
+        <tr>
+            <td>`onSetActiveIcon`</td>
+            <td>default</td>
+            <td>Fired to trigger a specific panel icon to show active state</td>
+            <td>
+                <dl>
+                    <dd>`panelNum`</dd>
+                    <dt>the number of the panel whose icon state needs to be set to active</dt>
+                </dl>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Methods
+
+| Method | Description | Parameters |
+|--------|-------------|------------|
+| `showIconPage` | Scrolls the icon container to make sure the current active icon is visible. | none |
+
+## Subcomponents
+
+This component has the following
+[subcomponents](http://docs.fluidproject.org/infusion/development/SubcomponentDeclaration.html):
+
+<table>
+    <tr><th>Name</th><th>Description</th><th>Values</th><th>Default</th></tr>
+    <tr>
+        <td>`icon`</td>
+        <td>This is a dynamic component. A new instance is created for each `onCreateIcon` event that is fired.</td>
```

Comment:
The second sentence probably can be improved to: A new instance is created when `onCreateIcon` event is fired.